src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -174,5 +174,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\$(AssemblyName)$(TargetExt)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName>StrongName</StrongName>
+    </FilesToSign>
+  </ItemGroup>
```

Comment:
Done in https://github.com/Microsoft/msbuild/pull/1078/commits/21042756dc84b7a8ddcc513e96e971023c37a08b

Let me know if it look good